examples/todoapp/linux/flutter/generated_plugins.cmake

```diff
@@ -4,6 +4,7 @@
 
 list(APPEND FLUTTER_PLUGIN_LIST
   flutter_secure_storage_linux
+  realm
```

Comment:
Not sure if we need this file committed.

Reply:
Added to .gitignore